plugins/communications-app/BodyForm/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Form } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import TextEditor from '@communications-app/src/components/bulk-email-tool/text-editor/TextEditor';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import messages from './messages';
+
+const BodyForm = () => {
+  const intl = useIntl();
```

Comment:
Why not directly destructure here:
 

```suggestion
  const { formatMessage } = useIntl();
```

Or is a standar for this kind of projects?

Reply:
It's not exactly a standard for all MFE (depending on the maintainer), but it's the common way to do it.

Reply:
Most of the MFE's use this structure principal because they use injectIntl to pass intl so I wanted to follow that way